pom.xml

```diff
@@ -160,12 +160,12 @@
 		<dependency>
 			<groupId>com.force.api</groupId>
 			<artifactId>force-partner-api</artifactId>
-			<version>34.0</version>
+			<version>36.0.0</version>
 		</dependency>
 		<dependency>
 			<groupId>com.force.api</groupId>
 			<artifactId>force-wsc</artifactId>
-			<version>34.0</version>
+			<version>36.1.1</version>
```

Comment:
These two are independent projects (http://mvnrepository.com/artifact/com.force.api/force-wsc , http://mvnrepository.com/artifact/com.force.api/force-partner-api) and I have used the latest versions from these (force-wsc has released 2 more versions since I developed this code; I shall change that)
Let me know if you think we need to maintain parity between these two dependencies